### PR TITLE
[Console] [formatTime] Considering minutes and returning a time converted to decimal

### LINKED ISSUE #58692: formatTime in Helper teturns a converted time
### Description

The formatTime function in Helper only considers the hour value without taking minutes into consideration and returns a converted time.

### Example

Even though the execution time is 2 hours and 50 minutes, this only returns 2 hours. This should return 2.8

This should display:

Execution Statistics
--------
Execution Time: 2.8 hours

Instead of:

Execution Statistics
--------
Execution Time: 2 hours

### PR DESCRIPTION
**Pull request by espoirbaraka**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Issues        | Fix #58692
| License       | MIT

### CODE REVIEW

**Comment by espoirbaraka** [comment]
This pull request is related to this issue : [issue](https://github.com/symfony/symfony/issues/58692)

**Comment by GromNaN** [comment]
Hello, thank you for contributing.

Could you add test cases to [`HelperTest`](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Console/Tests/Helper/HelperTest.php#L20) in order to show the bug you are fixing and ensure there is no regression in the future.

**Comment by espoirbaraka** [comment]
> Could you add test cases

I have already done this. I added 3 cases to show how it works

File: src/Symfony/Component/Console/Tests/Helper/HelperTest.php [hunk 1]
```diff
@@ -33,9 +33,12 @@ public static function formatTimeProvider()
             [121,    '2 mins, 1 sec', 2],
             [3599,   '59 mins, 59 secs', 2],
             [3600,   '1 hr', 2],
+            [5199,   '1 hr, 26 mins, 39 secs', 1.4],
```
Comment (GromNaN) [inline src/Symfony/Component/Console/Tests/Helper/HelperTest.php:36]: The 3rd parameter must be an `int` representing the `$precision`, i.e. the number of units to be displayed.

File: src/Symfony/Component/Console/Helper/Helper.php [hunk 2]
```diff
@@ -122,7 +129,24 @@ public static function formatTime(int|float $secs, int $precision = 1): string
             $secs -= $seconds;
         }
 
-        return implode(', ', array_reverse($times));
+        if (3600 === $format[0]) {
```
Comment (GromNaN) [inline src/Symfony/Component/Console/Helper/Helper.php:132]: The change must not be specific to a given time format.

**Review by GromNaN — CHANGES_REQUESTED** [review]
I'm closing because it looks like you're trying to implement a feature that already exists: choose precision. The implementation is not in the logic of the existing code.

**Pull request closed without merging**